code/mission/missionmessage.cpp

```diff
@@ -61,6 +63,16 @@ builtin_message Builtin_messages[] = {
   #undef X
 };
 
+#define BUILTIN_BOOST_LEVEL_ONE  1
+#define BUILTIN_BOOST_LEVEL_TWO  2
+#define BUILTIN_MATCHES_TYPE     4
+#define BUILTIN_MATCHES_FILTER   8
+#define BUILTIN_MATCHES_MOOD    16
+#define BUILTIN_MATCHES_SPECIES 32
+#define BUILTIN_MATCHES_PERSONA 64
```

Comment:
Aside from the new BOOST_LEVEL entries, the BUILTIN_MATHES_* constants were originally static const int. While I'm fine with them either way, there may be some additional safety by making them `constexpr` instead of a macro, in addition to being easier to work with in debuggers.

Reply:
Done.